src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -321,6 +324,7 @@ const wchar_t* mainWindowTitle = L"Endless Installer";
 
 #define ALL_FILES					L"*.*"
 
+#define CODING_INSTALLER_NAME    L"coding-installer.exe"
```

Comment:
Yep, I have to change this. Actually, I was thinking that it would be good to check for the `code-installer` prefix, and `.exe` suffix. The idea is that sometimes users can be a bit goofy, and if they download the same file twice by accident, they may end up with several files with names like:

* `code-installer.exe`
* `code-installer(1).exe`
* `code-installer.1.exe`

...and so on, depending on which browser or download manager they use to fetch the file. Any of the names above should trigger coding mode, IMO.